### ISSUE TITLE
Paging and filters required on Import Result page

### ISSUE THREAD

**Issue by ErenPhayte** [mylar3/mylar3#152]
for large comic collections, this page doesnt load. Paging is required on this page to limit the number of records at a time. Filtering options would be great so you can find comics to import or pull back only comics that havent been imported yet (status).

**Comment by barbequesauce** [comment]
Give that a shot...

**Comment by ErenPhayte** [comment]
Not sure what was done, but the page doesn't load at all.. been sitting waiting for over 30 minutes

I think the home page does a similar thing - pulls ALL the data, then only pages the results, instead of only pulling back say 20 records at a time.

These sections probably need an overhaul using AJAX to page results limited to 20 (or whatever the person had selected in the dropdown you have).

**Comment by barbequesauce** [comment]
Oh definitely - all of Mylar's tables are pulling the full result set and then paging in the browser's memory using dataTables.

PRs are welcome, if you have an idea on how to change them :)

**Comment by evilhero** [comment]
We do accept PR's. Until either myself or @barbequesauce has time to dedicate to this - it's probably not going to be a quick turnaround as the backend has to change to accomodate all the new Ajax methods that would be required.

**Comment by ErenPhayte** [comment]
I am not familiar with the code base, else I would add to it, but when I got a moment I will start familiarising myself and maybe help out - I have a big collection of comics so I am a perfect use case for this haha.

**Comment by ErenPhayte** [comment]
Uh, you are using Datatables for this. This shouldn't be too much of an overhaul.

https://datatables.net/examples/data_sources/ajax.html

I may spend some time on this.

**Comment by eroth312** [comment]
@evilhero I think this was fixed with my change last weekend. #248 
It's at least working for me now on the dev branch

**Comment by mylar3** [comment]
Yeah this is totally different than that PR. This involves pagination of the GUI, so if you have a very large dataset it doesn't have to load the entire thing to display - but with that comes a whole whack of other things that will be rolled out at the same time hopefully.

**Comment by barbequesauce** [comment]
Closing due to staleness, if this is still an issue please resubmit a new dare package with details please.
